src/xcode/ENA/ENA/Source/Scenes/Events/OnBehalfCheckinSubmission/Info/OnBehalfInfoViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+class OnBehalfInfoViewController: DynamicTableViewController, DismissHandling, FooterViewHandling {
+
+	// MARK: - Init
+
+	init(
+		onPrimaryButtonTap: @escaping () -> Void,
+		onDismiss: @escaping () -> Void
+	) {
+		self.onPrimaryButtonTap = onPrimaryButtonTap
+		self.onDismiss = onDismiss
+
+		super.init(nibName: nil, bundle: nil)
+	}
+
+	@available(*, unavailable)
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	// MARK: - Overrides
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+
+		if let parent = parent, !parent.isKind(of: UINavigationController.self) {
```

Comment:
Why is this happening? A little comment would be nice.

Reply:
That distinction was not really needed here, so I just removed it now